promptui/select.go

```diff
@@ -25,7 +25,7 @@ func (s *Select) Run() (int, string, error) {
 }
 
 func (s *Select) innerRun(starting int, top rune) (int, string, error) {
-	stdin := readline.NewCancelableStdin()
+	stdin := readline.NewCancelableStdin(os.Stdin)
```

Comment:
Yep -- the dep got upgraded.

Should we document dependency upgrades? It doesn't seem useful to anyone but those who contribute to the project.